### PR TITLE
Add adem's pin to map

### PR DESCRIPTION
**Pull request by p1v0t**
Adding KBU to the map!
closes #6784

### CODE REVIEW

File: _pins/adem.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubhandle: Adem
+latitude: 41.205997
+longitude: 32.659564
```
Comment (crichID) [inline _pins/adem.json]: @p1v0t you are so close! The only thing remaining is to add three dashes on the last line `---`. If you need any help, please check out this video: https://vimeo.com/201011288

**Comment by gerryhcss** [comment]
@p1v0t you need to remove the plus sign on the first line and add --- to last line

**Comment by crichID** [comment]
@gerryhcss oops, be careful not to close the pull request :wink: just comment!

**Comment by mitz840** [comment]
@Adem this looks good 🎉 . Just need to add ---  at the bottom of your code.

**Comment by adem** [comment]
next time please check before mentioning the wrong account

**Pull request merged**